dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/hooks/EventDataValuesValidationHook.java

```diff
@@ -59,6 +60,8 @@ public void validateEvent( ValidationErrorReporter reporter, Event event )
     {
         TrackerImportValidationContext context = reporter.getValidationContext();
 
+        ProgramStage programStage = context.getProgramStage( event.getProgramStage() );
```

Comment:
ProgramStage cannot be null because previous validators would fail if so.
We can add a checkNotNull here and then not checking anymore if it is null